src/tests/Store.jest.js

```diff
@@ -453,4 +483,63 @@ describe('Store', () => {
             ]);
         });
     });
+
+    describe('developer warnings', () => {
+        it('calls console.warn when a non-array selector is passed to getPartialState', () => {
+            const warn = console.warn = jest.fn();
```

Comment:
could do a spy instead `const warn = jest.spyOn(console, 'warn');` and still have same mock functions, but easier to clean up / restore:
```
warn.mockReset();
warn.mockRestore();
```